packages/cozy-client/src/types.js

```diff
@@ -160,7 +160,7 @@ import { QueryDefinition } from './queries/dsl'
  * @property {string} name - Name of the folder
  * @property {object} metadata - Metadata of the folder
  * @property {object} type - Type of the folder
- * @typedef {CozyClientDocument & FileDocument} IOCozyFolder - An io.cozy.files document
+ * @typedef {CozyClientDocument & FolderDocument} IOCozyFolder - An io.cozy.files document
```

Comment:
io.cozy.files document or io.cozy.folders document ???

Reply:
`io.cozy.files` is the doctype (i.e. the CouchDB « table ») while `file` and `folder` (`directory` in fact…) are two types of `io.cozy.files`.
Does that make sense?